tsconfig.json

```diff
@@ -8,6 +8,7 @@
     { "path": "./crates/codegen/runtime/npm/package/tsconfig.json" },
     { "path": "./crates/solidity/outputs/npm/package/tsconfig.json" },
     { "path": "./crates/solidity/outputs/npm/tests/tsconfig.json" },
+    { "path": "./crates/solidity/testing/perf/npm/tsconfig.json" },
```

Comment:
I wonder if it was possible to test this with a `benchmark` run from a private branch?
If so, I suggest posting a link to the bencher query/graph showing the results, so that we can review them in parallel.

Reply:
I'm confused about the place of this comment. This has nothing to do with this added line here, right?

Reply:
You're right. My bad. It should have been on the CLI command module.

Reply:
Copied to https://github.com/NomicFoundation/slang/pull/1325#discussion_r2100908082